Project.toml

```diff
@@ -5,17 +5,24 @@ version = "0.1.3"
 
 [deps]
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataFramesMeta = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Feather = "becb17da-46f6-5d3c-ad1b-1c5fe96bc73c"
+FeatherFiles = "b675d258-116a-5741-b937-b79f054b0542"
+Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
+InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptFormat = "f4570300-c277-12e8-125c-4912f86ce65d"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 ParameterJuMP = "774612a8-9878-5177-865a-ca53ae2495f9"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 PowerModels = "c36e90e8-916a-50a6-bd94-075b64ef4655"
 PowerSystems = "bcd98974-b02a-5e2f-9ee0-a103f5c450dd"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Why was Revise added to the project? Some of these packages are used for testing and development and don't make part of the main dependencies. Please remove them.